### DISCUSSION TITLE
Will Seaweedfs allow HDDs to sleep when nothing is accessing anything?

### DISCUSSION THREAD

**Discussion by strayer** [seaweedfs/seaweedfs#2671] [Q&A]
I want to add S3-compatible storage to an Unraid NAS that is only used for daily backups with restic. I’m running the restic-rest-server right now and the underlying disks have no problems to go into standby if no backup is running.

For various reasons I now want to move this to either Minio or seaweedfs, but I really need to make sure that disk are only spinning when restic is doing anything. 

Is this possible with seaweedfs? I wasn’t able to find any documentation whether seaweedfs might keep files open forever or periodically write something. 

I have a SD card in the system for frequently read but rarely written files like config files for Docker containers and such, if that helps in any way.

**Comment by chrislusf** [comment]
SeaweedFS does keep all volume files open, but there are no writes if there are no write requests or any admin script operations.

**Reply by strayer** [reply]
Thanks for responding! I did a quick test and it seems like the disks do spin up again immediately, so I assume seaweedfs wouldn't work for me in this case. :(

**Reply by flomo29** [reply]
I can admit that. I have an ARM odroid board with a SATA harddrive and after a couple of minutes it spins down. and when i open a file over the fuse mount i can hear that the hdds spinning up, gave me the data and the spinning down and are completly silent for a long time.

**Reply by vchirikov** [reply]
I want to use the s3 exact the same way (in k8s)
@strayer did you find the s3 storage with this feature? (p.s. [minio issue](https://github.com/minio/minio/discussions/17928))
